tensorflow/c/c_api.cc

```diff
@@ -2803,4 +2803,43 @@ TF_Buffer* TF_GetRegisteredKernelsForOp(const char* name, TF_Status* status) {
   }
   return ret;
 }
+
+// TF_Server functions ----------------------------------------------
+
+TF_Server::TF_Server(tensorflow::ServerInterface* server) : server(server) {}
+
+TF_Server* TF_NewServer(const void* proto, size_t proto_len,
+                        TF_Status* status) {
+  tensorflow::ServerDef server_def;
+  if (!server_def.ParseFromArray(proto, static_cast<int>(proto_len))) {
+    status->status = InvalidArgument("Unparseable ServerDef");
```

Comment:
Let's spell the error out in more detail: "Could not parse provided bytes into a tensorflow.ServerDef protocol buffer."